doc/source/whatsnew/v0.24.0.txt

```diff
@@ -112,7 +112,7 @@ Timezones
 Offsets
 ^^^^^^^
 
--
+- :class:`Tick` subclasses can no longer be created with the argument `normalize=True` (:issue:`21427`)
```

Comment:
double backticks here. I would be a little more informative (say for example Minute, Second).

Reply:
Will do.